frontend/src/metabase/search/components/SearchResult.unit.spec.js

```diff
@@ -23,6 +25,50 @@ function collection({
   return collection;
 }
 
+describe("SearchResult > Tables", () => {
+  it("tables with initial_sync_status='incomplete' are not clickable", async () => {
+    const table = { id: 1, display_name: "Table Name" };
+    const database = { id: 1, name: "Database Name" };
+    fetchMock.get("path:/api/table/1", table);
+    fetchMock.get("path:/api/database/1", database);
+    const result = {
+      model: "table",
+      name: "Incomplete Table",
+      initial_sync_status: "incomplete",
+      table_id: 1,
+      database_id: 1,
+      getUrl: () => `/table/1`,
+      getIcon: () => ({ name: "table" }),
+    };
+    const onClick = jest.fn();
+    renderWithProviders(<SearchResult result={result} onClick={onClick} />);
```

Comment:
Could you follow our FE unit test styleguide using `setup` function? [example](https://github.com/metabase/metabase/blob/234d8985482ff0e0f82f7953ad233dd2b50fcda5/enterprise/frontend/src/metabase-enterprise/whitelabel/components/MetabotToggleWidget/MetabotToggleWidget.unit.spec.tsx#L11-L18). I'd say adding setup only for these new tests would suffice, but it might even be better to refactor the whole suite (not necessary btw just want to reiterate).

[This is a good read](https://www.notion.so/metabase/Mocking-entity-API-endpoints-b3c2cc113c9844a0822783d05ca2d508) if you're curious.

Reply:
I've updated the tests to use a setup function, let me know if this is what you mean:
[42a04d9](https://github.com/metabase/metabase/pull/31651/commits/42a04d913070dd98f6b7409b3d1959b2463c6f19)

Reply:
Replied in https://github.com/metabase/metabase/pull/31651#pullrequestreview-1487933838.